config/kubernetes/sig-release/teams.yaml

```diff
@@ -306,18 +306,18 @@ teams:
         - jeremyrickard # subproject owner / Technical Lead
         - jlbutler # 1.25 RT Lead Shadow
         - justaugustus # subproject owner / Chair
-        # - justin-chizer # 1.25 Bug Triage Shadow
+        - justin-chizer # 1.25 Bug Triage Shadow
         - jyotimahapatra # 1.25 CI Signal Shadow
         - katcosgrove # 1.25 Comms Lead
         - kcmartin # 1.25 Docs Lead
-        # - krol3 # 1.25 Docs Shadow
+        - krol3 # 1.25 Docs Shadow
```

Comment:
```suggestion
        - krol # 1.25 Docs Shadow
```

Reply:
krol3 should be correct but the org membership got send to krol :D. I can open a PR to fix it